policy/aggregation_type.go

```diff
@@ -214,6 +265,38 @@ func (aggTypes AggregationTypes) IsValidForTimer() bool {
 	return true
 }
 
+// Quantiles returns all the quantiles found in the list aggregation types.
+func (aggTypes AggregationTypes) Quantiles(p pool.FloatsPool) []float64 {
+	var (
+		res         []float64
+		initialized bool
+		medianAdded bool
+	)
+	for _, aggType := range aggTypes {
+		q, ok := aggType.Quantile()
```

Comment:
Not really, this is not really an error, for example a policy contains [Min, Count, P9999] for a timer, then we need to only set 0.9999 in the stream bucket. So this is more like finding all the quantiles in a list of aggregation types, but the list is not expected to be all quantile types.